pyvex/block.py

```diff
@@ -477,6 +477,23 @@ def _is_defaultexit_direct_jump(self):
         target = self.default_exit_target
         return target is not None
 
+    def py_collect_exits(self):
```

Comment:
This should be private

Reply:
I will try to move this method to C instead.